gluster-block.spec.in

```diff
@@ -77,10 +77,14 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_unitdir}/gluster-blockd.service
 %{_unitdir}/gluster-block-target.service
 %else
-%attr(755, root, root) %{_initddir}/gluster-blockd
+%attr(0644,-,-) %{_initddir}/gluster-blockd
 %endif
 %config(noreplace) %{_sysconfdir}/sysconfig/gluster-blockd
-%{_sysconfdir}/sysconfig/gluster-block-caps.info
+%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block
+%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block/scripts
+     %attr(0755,-,-) %{_sharedstatedir}/gluster-block/scripts/wait-for-bricks.sh
+%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block/caps
+     %attr(0644,-,-) %{_sharedstatedir}/gluster-block/caps/gluster-block-caps.info
```

Comment:
What does this file contain? Generated/modified files (not editable by an admin) should be under `/var/lib/...`. If an admin may edit it, it should be under `/etc/gluster-block/...`. Non-editable defaults should be under `/usr/lib/gluster-block/...` with an optional overruling configuration in `/etc/gluster-block/...`.

Reply:
If there is an expectation that it get dynamically generated upon installation or running gluster-block, it should be under `/var/lib/gluster-block`. If the build process generates it, and the file stays unmodified as long as gluster-block is installed/running, place it in `/usr/lib/gluster-block`. In case gluster-blockd generates it every time it starts, then it should be `/run/gluster-block/` (that last one also needs a `/etc/tmpfiles.d/...` file for creating the directory on boot).

Reply:
Oh, and there probably is no need to have scripts/cap/.. subdirectories, just place the files directly under `${wherever}/gluster-block/` directly.

Reply:
Yes, lets use /var/lib/gluster-block for the caps, as it might be compile time generating file.

Assuming you want not to create a dir for scripts as well,
The new spec change will look like:
```
%dir %attr(0755,-,-) %{_libexecdir}/gluster-block                                                                   
      %attr(0755,-,-) %{_libexecdir}/gluster-block/wait-for-bricks.sh             
%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block                            
      %attr(0644,-,-) %{_sharedstatedir}/gluster-block/gluster-block-caps.info    
```

@nixpanic does this look good ?